### PR TITLE
Dm 3013

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-3013

## Description - what does this code do?
Displays all published/enabled/approved practices on an empty search

## Testing done - how did you test it/steps on how can another person can test it 
Make sure you see all published/enabled/approved practices, that the `Load more` button is working as intended, and the `Sort by most relevant` is disabled by doing the following:
1. Visit `/` and click on the header search icon.
2. Visit `/` and click on the Search button above the `Browse all innovations` link.
3. Visit `/` and click on `Browse all innovations` link.
4. Visit `/about`, open the "How do I use the Diffusion Marketplace?", and click on `browsing all innovations`.
5. Visit `/search`.

**_BONUS:_ Add search text and/or use filters to make sure everything is still working as intended.**

### _**Load/Response Times**_
I ran 10 trials using `loadsy.io` for both the master branch and this branch. I set the thread count to `50` and took the average response time for both branches on the DEV environment. I also manually tested the load times on DEV for both branches by clicking on various buttons/links from the repro steps above that take the user to an empty search page. I did 5 trials for this method. Here are the results:

_**Loadsy.io**_
1. Average response time for search page with no query (master branch, 50 thread count): `19.842s`.
2. Average response time for search page with no query (dm-3013 branch, 50 thread count): `19.349s`.

_**Manually visiting `/search`**_
1. Average search page load time on dev (master branch): `6.824s`.
2. Average search page load time on dev (dm-3013 branch): `6.163s`.

**Feel free to test this as well by comparing DEV to STG**

## Acceptance criteria
- Display all practices on empty search
- Display all practices on “Browse all innovations”
- Load by most adoptions
- Load 12 (I think, whatever our standard is) and then "Load more"
- Do this for the "About Us" section search 
- Check search load time using performance monitoring

## Definition of done
- [X] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by emyl3 — COMMENTED** [review]
one thing I noticed... let me know if it's not an issue :) Great work on this, Josh!

File: app/assets/stylesheets/dm/components/_search_no_results.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
   height: 40vh;
   margin-top: 40px;
 
-  .no-query-p, .no-query-results-p {
+  .no-query-results-p {
```
[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/assets/stylesheets/dm/components/_search_no_results.scss:6]: We do not use the `.no-query-p` on the visn pages. We only use the `.no-query-results-p`. If a visn does not have any created/adopted practices, we show the `.no-query-results-p`. The same can be said if the user searches for something and no results are returned. Here's an example from the master branch:

[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/assets/stylesheets/dm/components/_search_no_results.scss:6]: I will remove that line of code from the `visns/_show.js.erb`. Thanks Elisa!

[the remark this answers is not in the document]
Comment (emyl3) [reply app/assets/stylesheets/dm/components/_search_no_results.scss:6]: ahh gotcha! thank you! :)

**Review by emyl3 — APPROVED** [review]
Good job on this!

**Pull request merged**